### PR TITLE
Add Analysis Environment File

### PR DESCRIPTION
**Pull request by gwaygenomics**
Pull Request adds a new environment file and specifies install instructions in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,10 +70,26 @@ to current tools change.
...
 We use Jupyter notebooks and R scripts to visualize and summarize results.
 
+These analyses use a separate conda environment which is specified in
+[`analysis_environment.yml`](https://github.com/greenelab/pdx_exomeseq/blob/master/analysis_environment.yml).
+
+Follow these steps to install and begin using this conda environment:
+
+```bash
+conda env create --force --file analysis_environment.yml
+source activate pdx-exomeseq-analysis
```
Comment (dhimmel) [inline README.md]: r u no use conda recent v >= 4.4 conda activate pdx-exomeseq-analysis much improved more compatible

File: README.md [hunk 2]
```diff
@@ -70,10 +70,26 @@ to current tools change.
...
+
+# Setup Jupyter kernel
+python -m ipykernel install --user --name pdx-exomeseq-analysis --display-name pdx-analysis
```
Comment (dhimmel) [inline README.md]: I don't have to do this. Instead I use these lines in env:

```yml
  - conda-forge::ipykernel=4.8.1
  - conda-forge::nb_conda_kernels=2.1.0
```

File: analysis_environment.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+name: pdx-exomeseq-analysis
+channels:
+- r
+- bioconda
+- conda-forge
+dependencies:
+- anaconda::pandas=0.22.0
+- anaconda::plotnine=0.3.0
```
Comment (dhimmel) [inline analysis_environment.yml]: I'd switch anaconda over to conda-forge for these two. Always best to stay within channel. conda-forge preferred over anaconda

File: README.md (hunk 2 above)
Comment (gwaygenomics) [reply README.md]: nice, that did the trick

**Comment by gwaygenomics** [comment]
Thanks for conda upgrade tips @dhimmel - ready for rereview

**Pull request merged**